code/modules/clothing/suits/ego_gear/_ego_gear.dm

```diff
@@ -144,3 +158,7 @@
 	var/mob/living/carbon/human/H = usr
 	H.update_inv_wear_suit()
 	H.update_body()
+
+/datum/movespeed_modifier/too_many_armors
+	variable = TRUE
+	multiplicative_slowdown = 3.3 //Roughly 1/3 speed for holding too many armors
```

Comment:
This is fucking mental, change from 3.3 to like 0.8 or some shit. 0 is no change, 1 is about half speed